hphp/system/php/redis/Redis.php

```diff
@@ -1096,6 +1096,12 @@ protected function sockReadData(&$type) {
           if (!strncmp($line, '-ERR SYNC ', 10)) {
             throw new RedisException("Sync with master in progress");
           }
+          if (!strncmp($line, 'OOM ', 4)) {
+            throw new RedisException("OOM command not allowed when used memory > 'maxmemory'");
```

Comment:
@mofarrell sorry about the delay. For some reason, I did not get notification from github about your feedback. Just pushed a change for it.